spec/lib/contacts/distributed_lock_spec.rb

```diff
@@ -5,12 +5,12 @@
 RSpec.describe "Contacts::DistributedLock" do
   describe "#lock" do
     it "yields the block when a redis lock can be acquired" do
-      expect(Redis.current).to receive(:lock).and_call_original
+      expect_any_instance_of(Redis).to receive(:lock).and_call_original
       expect { |b| Contacts::DistributedLock.new.lock(&b) }.to yield_control
     end
 
     it "silently doesn't a yield the block when a lock can't be acquired" do
-      allow(Redis.current).to receive(:lock).and_raise(Redis::Lock::LockNotAcquired)
+      allow_any_instance_of(Redis).to receive(:lock).and_raise(Redis::Lock::LockNotAcquired)
```

Comment:
Hmmm, I'm surprised that Rubocop allowed this: https://www.rubydoc.info/gems/rubocop-rspec/1.7.0/RuboCop/Cop/RSpec/AnyInstance